### PR TITLE
Clean up project.jsons

### PR DESCRIPTION
**Pull request by blairconrad**
Inspired by a discussion in #928.

Am I crazy?

### CODE REVIEW

File: tests/FakeItEasy.Tests.netstd/project.json [hunk 1]
```diff
@@ -1,13 +1,7 @@
-{
+﻿{
```
Comment (adamralph) [inline tests/FakeItEasy.Tests.netstd/project.json]: FYI this is an encoding change from ASCII to UTF8-BOM, inline with Visual Studio's preference and reflected in our `.editorconfig`.

**Comment by blairconrad** [comment]
Thanks, @adamralph. Squashed.

**Comment by blairconrad** [comment]
Also, it looks like we could ditch `NETCORE` from the project.jsons altogether, with a small change to FakeOptionsBuilderTests.cs.

**Comment by blairconrad** [comment]
Okay. I'm done, unless someone wants something else. Sorry for the churn.

File: src/FakeItEasy.nuspec [hunk 2]
```diff
@@ -18,16 +18,6 @@
...
-        <dependency id="System.Console" version="4.0.0" />
-        <dependency id="System.Diagnostics.Tools" version="4.0.1" />
-        <dependency id="System.Diagnostics.Debug" version="4.0.11" />
-        <dependency id="System.IO.FileSystem" version="4.0.1" />
-        <dependency id="System.IO.FileSystem.Primitives" version="4.0.1" />
-        <dependency id="System.Linq.Expressions" version="4.1.0" />
-        <dependency id="System.Reflection" version="4.1.0" />
-        <dependency id="System.Reflection.Emit" version="4.0.1" />
-        <dependency id="System.Reflection.Extensions" version="4.0.1" />
-        <dependency id="System.Reflection.TypeExtensions" version="4.1.0" />
```
Comment (thomaslevesque) [inline src/FakeItEasy.nuspec]: This worries me a little... have you checked that the generated nupkg works as expected?

File: tests/FakeItEasy.IntegrationTests.netstd/project.json [hunk 3]
```diff
@@ -10,23 +10,15 @@
         "../FakeItEasy.IntegrationTests/**/*.cs"
       ]
     },
-    "define": [ "NETCORE" ]
-  },
-
-  "dependencies": {
-    "xunit.abstractions": "2.0.1-rc2",
-    "xunit.extensibility.core": "2.2.0-beta2-build3300",
-    "xunit.extensibility.execution": "2.2.0-beta2-build3300"
```
Comment (thomaslevesque) [inline tests/FakeItEasy.IntegrationTests.netstd/project.json]: I don't understand... without this, there isn't a single reference to XUnit. How can it work? I see that it builds, but I don't really like not understanding why :wink:

File: src/FakeItEasy.nuspec (hunk 2 above)
Comment (blairconrad) [inline src/FakeItEasy.nuspec]: No. Thanks for the poke.

File: tests/FakeItEasy.IntegrationTests.netstd/project.json (hunk 3 above)
Comment (blairconrad) [inline tests/FakeItEasy.IntegrationTests.netstd/project.json]: It gets it from FakeItEasy.Tests.netstd

File: src/FakeItEasy.nuspec (hunk 2 above)
Comment (blairconrad) [inline src/FakeItEasy.nuspec]: It works.

**Comment by adamralph** [comment]
Thanks @blairconrad !

**Comment by blairconrad** [comment]
Thanks for the merge, @adamralph, and the paranoia, @thomaslevesque!

File: tests/FakeItEasy.IntegrationTests.netstd/project.json (hunk 3 above)
Comment (thomaslevesque) [inline tests/FakeItEasy.IntegrationTests.netstd/project.json]: OK then ;)

**Pull request merged**